### PR TITLE
mgr/dashboard: Changed background colors for UI views, panel headers, panel footers, table headers

### PR DESCRIPTION
**Pull request by gnehapk**
Fixes: http://tracker.ceph.com/issues/35692

- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

Before -
background-color-before
panel-header-footer-before
table-header-before

After -
background-color-after
panel-header-footer-after
table-header-after

### CODE REVIEW

**Comment by gnehapk** [comment]
jenkins retest this please

File: src/pybind/mgr/dashboard/frontend/src/app/app.component.scss [hunk 1]
```diff
@@ -1,7 +1,6 @@
 @import '../defaults';
```
[the remark this answers is not in the document]
Comment (gnehapk) [reply src/pybind/mgr/dashboard/frontend/src/app/app.component.scss]: ack

**Comment by maceledo** [comment]
@LenzGr I agree, seeing it implemented, the gray background really dilutes the table styling. After discussion with Ju I have decided to propose reverting these pages back to white background color, while leaving the landing page / dashboard with the gray background. So essentially going back to your "before" screenshot above for all pages, but still keeping the landing page update as-is.

**Comment by julienlim** [comment]
@gnehapk @LenzGr @maceledo @epuertat 
+1 on leaving the other pages alone (i.e. "before" state).

It's okay to have the landing page have a different background color as we also have a similar treatment for other dashboard-like pages (see screenshot below) -- specifically, the landing page is unique and it's still consistent with the pages with grafana, and the other UI/pages also have gray that the transition is not jarring.

screen shot 2019-01-10 at 11 44 17 am

**Comment by LenzGr** [comment]
> +1 on leaving the other pages alone (i.e. "before" state).

Thanks for your feedback @julienlim - much appreciated. This is fine for me as well. So should we simply close this PR, or are there any changes remaining that should be kept?

**Comment by julienlim** [comment]
@LenzGr @gnehapk 

+1 on closing this PR as it does not seem like there are any changes remaining that needs to be kept.

**Comment by gnehapk** [comment]
Closing this PR since the changes present are no longer required now.

**Pull request closed without merging**